assert/assertions_test.go

```diff
@@ -637,7 +637,12 @@ func Test_samePointers(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			tt.assertion(t, samePointers(tt.args.first, tt.args.second))
+			same, ok := samePointers(tt.args.first, tt.args.second)
+			if !ok {
+				tt.assertion(t, false) // both arguments must be pointers
```

Comment:
You can't use `tt.assertion` for this, it might be expecting `false`.

Rename `assertion` to `same` and add another field called `ok` using the same pattern. Then add tests to the table which pass the three combinations of non-pointers.

Reply:
Got it! I've made the necessary changes